react-native-sdk/package.json

```diff
@@ -58,13 +58,12 @@
     "peerDependencies": {
         "@amplitude/react-native": "0.0.0",
         "@giphy/react-native-sdk": "0.0.0",
-        "@react-native/metro-config": "*",
         "@react-native-async-storage/async-storage": "0.0.0",
         "@react-native-clipboard/clipboard": "0.0.0",
         "@react-native-community/netinfo": "0.0.0",
         "@react-native-community/slider": "0.0.0",
         "@react-native-google-signin/google-signin": "0.0.0",
-        "react-native": "*",
+        "react-native": "~0.75.0",
```

Comment:
The version should be 0.0.0 and update_mobile_rnsdk_version should take care of the react-native version update to match ours.

Reply:
Also, update_dependecies script may need an update to match react-native and @react-native/metro-config versions.

Reply:
I could make the script do it, but it will need this shape. If an app is using 0.75.1 that should be fine because they are compatible API wise.

As for the metro config, this SDK doesn't use it, it's really the app using the SDK that needs to, and since there is no code dependency there is no need for a peer dep.